### ISSUE TITLE
Exclude _specific_ tags/notes from the graph

### ISSUE THREAD

**Issue by ayberk** [foambubble/foam#932]
It'd be really nice if we could exclude notes with certain tags excluded from the graph. My use case for this would be the daily notes. My daily note template has the `#daily` tag, and my graph would be much cleaner if I could exclude all my daily notes.

Another option is to exclude a whole filter, which would achieve the same goal for me but it'd be much less flexible.

**Comment by riccardoferretti** [comment]
I see what you mean, that is currently not possible.
As a workaround, you can filter notes by type in the graph. You can defined a type for your note in its front matter.
More options might be available via the Foam API, to be released _in beta_ soon

**Comment by tv42** [comment]
I would like to exclude notes by pathname. I import 10,000 notes and the graph is.. shall we say "unworkable". I don't really want to add YAML frontmatter setting a node type for all of those files, either.

**Comment by riccardoferretti** [comment]
I completely agree with that, it's only been a matter of time to give the graph some extra love.
Beside some workarounds here and there, the work required to properly upgrade the graph experience is not super trivial.
If someone is interested in working on it, I will be happy to provide guidance

**Comment by theAkito** [comment]
To give this more context, I would like to give an example, how this issue affects not only bulk imports or otherwise extraordinary circumstances, i.e. making it seem like this is a rare issue.

I use the [Local History Visual Studio Code Extension](https://github.com/zabel-xyz/local-history), which naturally generates a lot of file, over time. All these files are shown in the graph, too. So, this issue affects even normal usage, with a medium amount of documents in a Foam workspace, if that particular user uses any extension, which generates a lot of files in that way. Additionally, even if the amount of files would be small, it would still be an issue, as they shouldn't be visible in the graph, at all, anyway.

That's just an FYI.

**Comment by riccardoferretti** [comment]
This should now be possible in Foam. Both by passing filters to the `show-graph` command, or by creating groups within the graph
